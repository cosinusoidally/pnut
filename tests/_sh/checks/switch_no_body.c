// expect_comp_failure
void main() {
  switch (1) putchar('a');
}
