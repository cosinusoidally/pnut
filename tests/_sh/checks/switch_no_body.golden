tests/_sh/checks/switch_no_body.c:5:0  comp_statement: switch without body
